# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1541695479 28800
#      Thu Nov 08 08:44:39 2018 -0800
# Node ID fb27bd81fb470e5d74c8961bc80c50ccdd4ba53d
# Parent  920a4ad8043b49201ec63f3bf6684d7cadf845c6
octave-svgconvert: delete unused #include <QtMath> (bug #54984)

* octave-svgconvert.cc: Delete unused #include <QtMath>, restore building
with Qt 4.x.

diff --git a/src/octave-svgconvert.cc b/src/octave-svgconvert.cc
--- a/src/octave-svgconvert.cc
+++ b/src/octave-svgconvert.cc
@@ -17,17 +17,16 @@ You should have received a copy of the G
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
 #include <iostream>
 
 #include <QtCore>
-#include <QtMath>
 #include <QtXml>
 
 #include <QApplication>
 #include <QFontDatabase>
 #include <QImage>
 #include <QPainter>
 #include <QPrinter>
 #include <QRegExp>
